src/Plugin/Block/HomeWelcomeBlock.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\localgov_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Block\BlockPluginInterface;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a 'Powered by LocalGovDrupal' block.
+ *
+ * @Block(
+ *   id = "localgov_home_welcome_block",
+ *   admin_label = @Translation("Home welcome block")
+ * )
+ */
+class HomeWelcomeBlock extends BlockBase implements BlockPluginInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return ['label_display' => FALSE];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+
+    $config = $this->getConfiguration();
+
+    if (!empty($config['localgov_home_welcome_message'])) {
+      $message = $config['localgov_home_welcome_message'];
+    }
+    else {
+      $message = $this->t('<h2>You have just installed a LocalGov Drupal site</h2>');
+    }
+
+    return [
+      '#markup' => $message['value'],
```

Comment:
This is throwing the following PHP exception following a fresh install: "Error: Cannot use object of type Drupal\Core\StringTranslation\TranslatableMarkup as array in Drupal\localgov_core\Plugin\Block\HomeWelcomeBlock->build() (line 41 of modules/contrib/localgov_core/src/Plugin/Block/HomeWelcomeBlock.php)." :(  Replacing "$message['value']" with "$message" fixes this.

Reply:
Thanks Adnan, that exception is fixed now.

> Because this block is tied to the localgov install profile, perhaps that would be a better home for this block rather than localgov_core module.

Ah, I wasn't aware install profiles could include anything other than config and translations - it did feel odd to have to put this in localgov_core. I'll move it over to the install profile and delete this branch, that makes much more sense.

Reply:
Thanks Dan.  Just noticed that @finnlewis has also written "This can sit in the locagov install profile." in [Simple home page - place holder](https://github.com/localgovdrupal/localgov/issues/96).  So there is now a stronger case for moving this code to the localgov install profile.